packages/react-components/react-drawer/src/components/DrawerHeaderTitle/renderDrawerHeaderTitle.tsx

```diff
@@ -0,0 +1,22 @@
+import * as React from 'react';
+import { getSlots } from '@fluentui/react-utilities';
+import type { DrawerHeaderTitleState, DrawerHeaderTitleSlots } from './DrawerHeaderTitle.types';
+import { DialogTitle, DialogTitleProps } from '@fluentui/react-dialog';
+
+/**
+ * Render the final JSX of DrawerHeaderTitle
+ */
+export const renderDrawerHeaderTitle_unstable = (state: DrawerHeaderTitleState) => {
+  const { slots, slotProps } = getSlots<DrawerHeaderTitleSlots>(state);
+
+  const titleProps = {
+    ...state.title,
+    action: slotProps.action,
+  } as DialogTitleProps;
+
+  return (
+    <slots.root {...slotProps.root}>
```

Comment:
This is odd to me, why do you need to render the DialogTitle within another wrapper/layout element?

Reply:
Also there is an assumption that the DialogTitle component will be rendered inside a DialogBody which declares all the gridareas

Reply:
This was fixed by using composition.